common/main/fil.xml

```diff
@@ -2331,61 +2331,61 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
 							<dayPeriod type="night1">gabi</dayPeriod>
 						</dayPeriodWidth>
 						<dayPeriodWidth type="narrow">
-							<dayPeriod type="midnight">↑↑↑</dayPeriod>
+							<dayPeriod type="midnight">hatinggabi</dayPeriod>
 							<dayPeriod type="am">am</dayPeriod>
-							<dayPeriod type="noon">↑↑↑</dayPeriod>
+							<dayPeriod type="noon">tanghaling-tapat</dayPeriod>
 							<dayPeriod type="pm">pm</dayPeriod>
 							<dayPeriod type="morning1">umaga</dayPeriod>
-							<dayPeriod type="morning2">↑↑↑</dayPeriod>
+							<dayPeriod type="morning2">madaling-araw</dayPeriod>
 							<dayPeriod type="afternoon1">sa hapon</dayPeriod>
 							<dayPeriod type="evening1">sa gabi</dayPeriod>
-							<dayPeriod type="night1">↑↑↑</dayPeriod>
+							<dayPeriod type="night1">gabi</dayPeriod>
```

Comment:
There are 4 errors in fil.xml, all for the value "gabi"

43 errors in bs-Cyrl plus 4 errors in fil = total 47 errors for this PR